resources/views/includes/status.blade.php

```diff
@@ -55,7 +55,7 @@
                 </li>
```

Comment:
<img width="884" alt="Screenshot 2021-04-17 at 10 59 13" src="https://user-images.githubusercontent.com/4103693/115107567-f3785a00-9f6b-11eb-9390-65ba5ad8f84e.png">
<img width="802" alt="Screenshot 2021-04-17 at 10 59 37" src="https://user-images.githubusercontent.com/4103693/115107572-01c67600-9f6c-11eb-817b-1398755b87a5.png">
The Status Cards are now barely silhouetted against the background. It all just looks white.

Reply:
Idk, I'm kinda fine with that. Should i resolve this conversation or add another class? 😅

Reply:
Would you be okay with leaving it the way it is? If yes, could you please approve it so I can merge it? Thanks! 😊

Reply:
![image](https://user-images.githubusercontent.com/1267894/116370115-5e4a5080-a80a-11eb-81d3-9c3498612bd6.png)